client/src/components/Map.tsx

```diff
@@ -93,11 +95,13 @@ const MapComponent: FC<MapComponentProps> = (props) => {
                     });
                   }}
                 >
-                  <img
-                    src={`/document-${doc.type}-icon.png`}
-                    alt="Custom Marker"
-                    style={{ width: "4vh", height: "4vh" }}
-                  />
+                  {
+                    <img
+                      src={`/document-${doc.type}-icon.png`}
+                      alt="Custom Marker"
+                      style={{ width: "4vh", height: "4vh" }}
+                    />
+                  }
```

Comment:
delete parenthesis,

Reply:
Don't put styles in the components. Put that in the scss file